internal/webhook/v1alpha/conduit_webhook.go

```diff
@@ -218,7 +219,11 @@ func (v *ConduitCustomValidator) validateConnectors(cc []*v1alpha.ConduitConnect
 
 	fp := field.NewPath("spec").Child("connectors")
 	for _, c := range cc {
-		for _, fn := range connectorValidators {
+		if paramErr := validator.ValidateConnectorParameters(c, fp); paramErr != nil {
```

Comment:
My recommendation will be to make this a service and attaching it to the validator https://github.com/ConduitIO/conduit-operator/blob/main/internal/webhook/v1alpha/conduit_webhook.go#L152

Reply:
And it can be initialized as part of the boot up.